### ISSUE TITLE
Anti-Netrin-1 receptors DCC (deleted in colorectal carcinoma) antibody

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10548]
**Preferred term label:**

**Synonyms**
Anti-DCC

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against Netrin-1 receptors DCC (deleted in colorectal carcinoma)

PMID: 28251919 Antibodies against Netrin-1 receptors (DCC and UNC5a) and Caspr2 often coexist and associate with thymoma in patients with neuromyotonia and myasthenia gravis. 

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:589 Myasthenia gravis 

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000881
- label: Anti-DCC netrin 1 receptor antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:28251919
- def: The presence of autoantibodies in the serum that react against Netrin-1 receptors DCC (deleted in colorectal carcinoma).
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10548
- synonyms:
